docs/docs/monitoring/load-testing-guidelines.mdx

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_label: Load Testing Guidelines
+title: Load Testing Guidelines
+description: |
+  Information about how best to scale up your bot to support more parallel user activity
+  and how you can use tracing to help debug issues.
+---
+
+:::note
+
+Our team is currently in the process of running more extensive performance related tests, more information will be added here as we progress.
+
+:::
+
+## Overview
+
+In the process to gather metrics on our system's ability to handle increased loads and users we have performed tests to evaluate the maximum number of concurrent users a Rasa chatbot can handle with certain machine configurations.
+In each test case we spawned the following number of concurrent users at peak concurrency using a spawn rate of 1000 users per second. In our tests we used Rasa's [Http-Api](https://rasa.com/docs/rasa/pages/http-api),
```

Comment:
Can we add a definition first of what a spawn rate is, or link to some established glossary online?

Reply:
Linked a locust configuration page with a definition of spawn rate mentioned but nothing specific.